### PR TITLE
Update to allow Symfony 4 and fix CS

### PR DESCRIPTION
**Pull request by NicolasDievart**

### CODE REVIEW

File: Builders/FormTreeBuilder.php [hunk 1]
```diff
@@ -84,17 +84,17 @@ private function createNodeFromView(FormView $view)
...
-    	if (!isset($view->vars['compound']) || !$view->vars['compound']) {
+        if (!isset($view->vars['compound']) || !$view->vars['compound']) {
             return false;
-    	}
+        }
 
-    	foreach ($view->vars['block_prefixes'] as $prefix) {
-            if (in_array($prefix, $this->noChildren)) {
+        foreach ($view->vars['block_prefixes'] as $prefix) {
+            if (\in_array($prefix, $this->noChildren, true)) {
```
Comment (chalasr) [inline Builders/FormTreeBuilder.php]: Making it strict now is likely to cause obscure troubles, we should either keep it is as is or trigger a deprecation notice when a non strict match occur

Comment (chalasr) [reply Builders/FormTreeBuilder.php]: (`in_array` keeps optimized without the third argument)

Comment (NicolasDievart) [reply Builders/FormTreeBuilder.php]: You're right, I'll remove the third parameter

**Comment by Seb33300** [comment]
ping @chalasr @benji07 @Tom32i

**Pull request merged**